packages/sources/snowflake/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "@chainlink/snowflake-adapter",
+  "version": "1.0.4",
```

Let's start it on v1.0.0

```suggestion
  "version": "1.0.0",
```